src/components/Playlist.js

```diff
@@ -37,5 +37,8 @@ export default function Playlist({songs, deleteSong, addSong, mode}) {
 
 Playlist.propTypes = {
   songs:PropTypes.arrayOf(PropTypes.object).isRequired,
+  deleteSong:PropTypes.func,
+  addSong:PropTypes.func,
```

Comment:
I like that you've renamed addSong to addSongToPlaylist elsewhere - I think that is more descriptive. Is the idea here that it's not renamed addSongToPlaylist because we're in the Playlist component so that's sort of implicit?

Reply:
Yeah that was kind of my train of thought but I can definitely change it here too for the sake of consistency if you think it'd be a good idea!

Reply:
And for the error about the unique time window, yes that's because we already have the current show's playlist at that time. If you want to try adding a new playlist you can change the current show and/or date